### PR TITLE
Updated the periodic rate of can devices to limit usage of can network

### PR DESCRIPTION
**Pull request by tsieben**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Feedback.java [hunk 1]
```diff
@@ -15,6 +16,11 @@ public class Feedback extends SubsystemBase {
 
   public Feedback(int candleID) {
     bracelet = new CANdle(candleID);
+    bracelet.setStatusFramePeriod(CANdleStatusFrame.CANdleStatusFrame_Status_1_General, 0);
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/subsystems/Feedback.java:19]: Why 0?

File: src/main/java/frc/robot/subsystems/Wrist.java [hunk 2]
```diff
@@ -43,6 +44,15 @@ public Wrist(int WRIST_ID, float WRIST_HIGH_LIM, float WRIST_LOW_LIM) {
 
     wristNeo.enableSoftLimit(SoftLimitDirection.kForward, true);
     wristNeo.enableSoftLimit(SoftLimitDirection.kReverse, true);
+
+    wristNeo.setPeriodicFramePeriod(PeriodicFrame.kStatus0, 65535);
+    wristNeo.setPeriodicFramePeriod(PeriodicFrame.kStatus1, 65535);
+    wristNeo.setPeriodicFramePeriod(PeriodicFrame.kStatus2, 65535);
+    wristNeo.setPeriodicFramePeriod(PeriodicFrame.kStatus3, 65535);
+    wristNeo.setPeriodicFramePeriod(PeriodicFrame.kStatus4, 65535);
+    wristNeo.setPeriodicFramePeriod(PeriodicFrame.kStatus5, 65535);
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/subsystems/Wrist.java:53]: Status frame 5 has the absolute encoder data on it. Is that necessary for accessing the position in `isAtTarget`? Or does this work without it? I think that's the only status frame we _might_ need

**Review by ArchdukeTim — APPROVED** [review]

File: src/main/java/frc/robot/subsystems/Wrist.java (hunk 2 above)
Comment (tsieben) [reply src/main/java/frc/robot/subsystems/Wrist.java]: Yes, I updated that status rate back to the default.

File: src/main/java/frc/robot/subsystems/Elevator.java [hunk 3]
```diff
@@ -80,6 +81,9 @@ public Elevator() {
     elevNeoMotor1.setInverted(false);
     elevNeoMotor2.follow(elevNeoMotor1, true);
 
+    Util.setRevStatusRates(elevNeoMotor1, 5, 20, 20, 65535, 65535, 500, 65535, 250);
+    Util.setRevStatusRates(elevNeoMotor2, 100, 500, 500, 65535, 65535, 500, 65535, 250);
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/subsystems/Elevator.java:85]:
```suggestion
    Util.setRevStatusRates(elevNeoMotor1, 5, 20, 20, 65535, 65535, 65535, 65535, 65535);
    Util.setRevStatusRates(elevNeoMotor2, 500, 65535, 65535, 65535, 65535, 65535, 65535, 65535);
```

elevNeoMotor2 is just a follower so we shouldn't need anything from it

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 4]
```diff
@@ -25,6 +26,9 @@ public Intake(int intakeTopId, int intakeBottomId, double intakeSpeed, double ou
     intakeTop.restoreFactoryDefaults();
     intakeBottom.restoreFactoryDefaults();
 
+    Util.setRevStatusRates(intakeTop, 100, 65535, 500, 65535, 65535, 500, 65535, 250);
+    Util.setRevStatusRates(intakeBottom, 100, 65535, 500, 65535, 65535, 500, 65535, 250);
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/subsystems/Intake.java:30]:
```suggestion
    Util.setRevStatusRates(intakeTop, 100, 65535, 65535, 65535, 65535, 65535, 65535, 65535);
    Util.setRevStatusRates(intakeBottom, 100, 65535, 65535, 65535, 65535, 65535, 65535, 65535);
```

File: src/main/java/frc/robot/subsystems/Wrist.java [hunk 5]
```diff
@@ -43,6 +44,8 @@ public Wrist(int WRIST_ID, float WRIST_HIGH_LIM, float WRIST_LOW_LIM) {
 
     wristNeo.enableSoftLimit(SoftLimitDirection.kForward, true);
     wristNeo.enableSoftLimit(SoftLimitDirection.kReverse, true);
+
+    Util.setRevStatusRates(wristNeo, 10, 20, 20, 65535, 65535, 100, 65535, 250);
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/subsystems/Wrist.java:48]:
```suggestion
    Util.setRevStatusRates(wristNeo, 10, 20, 20, 65535, 65535, 100, 65535, 65535);
```

File: src/main/java/frc/robot/subsystems/chassis/NeoModule.java [hunk 6]
```diff
@@ -126,7 +126,8 @@ public void initialize() {
     drive.setIdleMode(config.driveIdleMode);
     steer.setIdleMode(config.steerIdleMode);
 
-    steer.setPeriodicFramePeriod(PeriodicFrame.kStatus5, 20);
+    Util.setRevStatusRates(steer, 65535, 20, 20, 65535, 65535, 20, 65535, 250);
+    Util.setRevStatusRates(drive, 65535, 20, 20, 65535, 65535, 65535, 65535, 250);
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/subsystems/chassis/NeoModule.java:130]:
```suggestion
    Util.setRevStatusRates(steer, 65535, 20, 20, 65535, 65535, 20, 65535, 65535);
    Util.setRevStatusRates(drive, 65535, 20, 20, 65535, 65535, 65535, 65535, 65535);
```

**Review by ArchdukeTim — CHANGES_REQUESTED** [review]

**Comment by ArchdukeTim** [comment]
Status frame 7 only has the IAccum value it seems https://docs.revrobotics.com/brushless/spark-max/revlib/firmware-changelog#breaking-changes. I don't think we need it

File: src/main/java/frc/robot/subsystems/Feedback.java [hunk 7]
```diff
@@ -15,6 +16,11 @@ public class Feedback extends SubsystemBase {
...
+    bracelet.setStatusFramePeriod(CANdleStatusFrame.CANdleStatusFrame_Status_4_ControlTelem, 0);
+    bracelet.setStatusFramePeriod(CANdleStatusFrame.CANdleStatusFrame_Status_5_PixelPulseTrain, 0);
+    bracelet.setStatusFramePeriod(CANdleStatusFrame.CANdleStatusFrame_Status_6_BottomPixels, 0);
+    bracelet.setStatusFramePeriod(CANdleStatusFrame.CANdleStatusFrame_Status_7_TopPixels, 0);
```
Comment (tsieben) [inline src/main/java/frc/robot/subsystems/Feedback.java:23]: Should update these to 255 as 0 may set it to 1ms

```suggestion
    bracelet.setStatusFramePeriod(CANdleStatusFrame.CANdleStatusFrame_Status_1_General, 255);
    bracelet.setStatusFramePeriod(CANdleStatusFrame.CANdleStatusFrame_Status_4_ControlTelem, 255);
    bracelet.setStatusFramePeriod(CANdleStatusFrame.CANdleStatusFrame_Status_5_PixelPulseTrain, 255);
    bracelet.setStatusFramePeriod(CANdleStatusFrame.CANdleStatusFrame_Status_6_BottomPixels, 255);
    bracelet.setStatusFramePeriod(CANdleStatusFrame.CANdleStatusFrame_Status_7_TopPixels, 255);
```

**Review by ArchdukeTim — APPROVED** [review]
LGTM

**Pull request merged**